### PR TITLE
fix(backend): add brackets if metadata service host is ipv6. Part of #5605

### PR DESCRIPTION
**Pull request by vsk2015**
**Description of your changes:**

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

**Comment by vsk2015** [comment]
@googlebot I signed it!

**Comment by Bobgy** [comment]
/lgtm
/approve

Thank you!

File: backend/metadata_writer/src/metadata_helpers.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def connect_to_mlmd() -> metadata_store.MetadataStore:
         'METADATA_GRPC_SERVICE_SERVICE_PORT', 8080))
 
     mlmd_connection_config = metadata_store_pb2.MetadataStoreClientConfig(
-        host=metadata_service_host,
+        host=[metadata_service_host] if isIPv6(metadata_service_host) else metadata_service_host,
```
Comment (Ark-kun) [inline backend/metadata_writer/src/metadata_helpers.py:41]: I'm not sure `MetadataStoreClientConfig` `host` accepts lists.

**Pull request merged**